examples/va/nfv/ovs-dpdk/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+components:
+  - ../../../../va/nfv/sriov/
```

Comment:
This is pointing to the SRIOV VA and needs to point to an underlying DPDK VA.  See [1] for SRIOV and then perhaps use that to inspire something similar for DPDK.

[1] https://github.com/openstack-k8s-operators/architecture/tree/main/va/nfv/sriov

Reply:
Ack. Thanks will update it.

Reply:
> Ack. Thanks will update it.

done

Reply:
You've changed the path, but the path does not actually exist.  You will need to create it and put the appropriate kustomization logic in it.  For example, see what we did for SRIOV [1].

[1] https://github.com/openstack-k8s-operators/architecture/tree/main/va/nfv/sriov

Reply:
@abays Ack. will create it. Thanks.

Reply:
@abays done.